### PR TITLE
more 0.12/0.13 things

### PR DESCRIPTION
**Pull request by Creeperface01**

### CODE REVIEW

File: src/pocketmine/block/AcaciaDoor.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Honza
+ * Date: 4. 11. 2015
+ * Time: 14:59
+ */
```
Comment (PEMapModder) [inline src/pocketmine/block/AcaciaDoor.php]: Remove these.

File: src/pocketmine/block/Door.php [hunk 2]
```diff
@@ -235,62 +235,62 @@ public function place(Item $item, Block $block, Block $target, $face, $fx, $fy,
...
-		$this->getLevel()->setBlock($this, new Air(), true);
+		$this->getLevel()->setBlock($this, new Air(), \true);
 
-		return true;
+		return \true;
 	}
 
-	public function onActivate(Item $item, Player $player = null){
+	public function onActivate(Item $item, Player $player = \null){
 		if(($this->getDamage() & 0x08) === 0x08){ //Top
```
Comment (PEMapModder) [inline src/pocketmine/block/Door.php]: what?

Comment (Creeperface01) [inline src/pocketmine/block/Door.php]: so i dont have to remove it right?

**Comment by Creeperface01** [comment]
done

**Pull request closed without merging**